meta-resin-common/recipes-connectivity/dnsmasq/files/resolv.conf

```diff
@@ -1,2 +1 @@
-# we use dnsmasq at 127.0.0.2 so that user containers can run their own dns cache and forwarder and not conflict with dnsmasq on the host
```

Comment:
Will this cause some changes in the behaviour of user containers?

Reply:
No. It should now.

Reply:
What if the user runs something in the container on 127.0.0.1? This was the point for which we have been using 127.0.0.2 here

Reply:
That was already a requirement for users to not user it, isn't it?

Reply:
We had this problem when we did this change to 127.0.0.2
Any particular reason now to switch back to 127.0.0.1 ?

Reply:
Decided in https://github.com/resin-os/resinos/issues/408#issuecomment-348046741